packages/grafana-ui/src/components/Select/Select.story.tsx

```diff
@@ -17,10 +18,52 @@ export default {
     docs: {
       page: mdx,
     },
+    knobs: {
+      disable: true,
+    },
+    controls: {
+      exclude: [
+        'getOptionValue',
+        'getOptionLabel',
+        'formatCreateLabel',
+        'filterOption',
+        'className',
+        'components',
+        'defaultValue',
+        'id',
+        'inputId',
+        'onBlur',
+        'onChange',
+        'onCloseMenu',
+        'onCreateOption',
+        'onInputChange',
+        'onKeyDown',
+        'onOpenMenu',
+        'prefix',
+        'renderControl',
+        'options',
+        'isOptionDisabled',
+        'maxVisibleValues',
```

Comment:
I think we can add `aria-label`, 'value' to this list too. Maybe the `noOptionsMessage` can be added as well as we don't have a story with no items to use it in, WDYT?

Reply:
yeah...I agree. Will do that right away